### PR TITLE
Feature#12 frontend

### PR DESCRIPTION
**Pull request by jnegrete15**
Set styles in:
 - Home page
 - Edit user
 - New task
 - Edit task

Screenshot from 2021-05-18 10-55-06
Screenshot from 2021-05-18 10-55-23
Screenshot from 2021-05-18 10-56-22
Screenshot from 2021-05-18 15-03-16

### CODE REVIEW

File: app/assets/stylesheets/users.scss [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+      .btn-update-user{
+        margin-left: 30px;
+        padding: 5px;
+      }
+    }
+
+    
+  }
+}
```
Comment (OchoaHub) [inline app/assets/stylesheets/users.scss:41]: I like your style

File: config/database.yml [hunk 2]
```diff
@@ -17,13 +17,21 @@
 default: &default
   adapter: postgresql
   encoding: unicode
+  username: <%= ENV['DB_USERNAME'] %>
+  password: <%= ENV['DB_PASSWORD'] %>
+  host: <%= ENV['DB_HOST'] %>
+  port: 5432
+  host: '172.20.0.2'
+  username: root
+  password: toor
```
Comment (OchoaHub) [inline config/database.yml:26]: Ty 4 the password

**Review by OchoaHub — APPROVED** [review]
LGTM

**Review by LPMendoza — APPROVED** [review]
Good job man!, I will recommend you this page to get inspired while designing interfaces

**Pull request merged**